raiden/api/v1/encoding.py

```diff
@@ -327,3 +327,43 @@ class ConnectionsLeaveSchema(BaseSchema):
     class Meta:
         strict = True
         decoding_class = dict
+
+
+class EventTransferSchema(BaseSchema):
+    payment_network_identifier = fields.Str(missing=None)
+    token_network_identifier = fields.Str(missing=None)
+    channel_identifier = fields.Str(missing=None)
```

Comment:
we agreed to not expose the identifiers, remove them.

Reply:
Thought that I would leave them in case someone will need them in the future, so that if they wanna add them to the the json response they could only add them to the fields. But I got the point and I removed them.

 Also should I remove the event identifier from the EventTransferSchema or can I just leave that, in case we might need it? I mention that it's not exposed through the API.

Reply:
> Also should I remove the event identifier from the EventTransferSchemas or can I just leave that, in case we might need it?

Remove it, it's not used.